packages/istanbul-lib-instrument/src/visitor.js

```diff
@@ -420,7 +420,7 @@ function coverIfBranches(path) {
     if (ignoreElse) {
         this.setAttr(n.alternate, 'skip-all', true);
     } else {
-        this.insertBranchCounter(path.get('alternate'), branch, n.loc);
+        this.insertBranchCounter(path.get('alternate'), branch);
```

Comment:
If I'm understanding correctly, does this end up using the column position of the `alternate` node, whereas before it was the start of the `if`?

Reply:
exactly, 
before: the line and column was both at the start of the if. 
after: line+col at the else. If no else/next alternate, then result is alternate locations[1]: {start: {}, end: {}}
thank you!!